query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/select/Select.java

```diff
@@ -146,6 +146,8 @@ default Select orderBy(@NonNull String columnName, @NonNull ClusteringOrder orde
     return orderBy(CqlIdentifier.fromCql(columnName), order);
   }
 
+  @NonNull
+  Select orderBy(@NonNull Ann ann);
```

Comment:
I also agree to save DESC ordering for later because:
1. I find it weird to add a feature that does not work yet and we cannot even test
2. Neither Bret nor I found relevant doc saying they may support DESC of vector search. @lukasz-antoniak did you find any? I find it hard to imagine in what cases we want to find the approximate farthest neighbor...?
3. If we want to add DESC later, we just need to add another function overload `Select orderByAnnOf(String columnName, CqlVector ann, ClusteringOrder order);`. I assume this is not hard.